provider/README.md

```diff
@@ -9,7 +9,7 @@ Searches for a provider in this order:
 1. `providerArg`, passed in via CLI
 2. Environment variable `SUMMON_PROVIDER`
 3. Executable in `/usr/local/lib/summon`
-(or `%ProgramW6432%\Cyberark Conjur\Summon\Providers` on Windows).
+(or `%ProgramW6432%\Cyberark Conjur\Summon\Providers` and `<path_to_summon_excutable>\Providers`  on Windows).
```

Comment:
We should clearly indicate that we check the system dir first and then the relative-to-executable one.